package.json

```diff
@@ -42,7 +42,8 @@
     "mocha": "^9.2.2",
     "nyc": "^15.1.0",
     "prettier": "^2.6.2",
-    "semantic-release": "^19.0.2"
+    "semantic-release": "^19.0.2",
+    "sinon": "^15.0.3"
```

Comment:
is this new dependency really necessary?

Reply:
yes, in order to mock some methods of the fs (filesystem) node module.